apps/prairielearn/src/sync/fromDisk/assessments.ts

```diff
@@ -287,6 +287,7 @@ function getParamsForAssessment(
     group_max_size: assessment.groupMaxSize || null,
     group_min_size: assessment.groupMinSize || null,
     student_group_create: !!assessment.studentGroupCreate,
+    student_group_name: assessment.studentGroupName ?? true,
```

Comment:
Nit: want to update the surrounding `student_group_*` values with nullish coalescing to match?